corehq/celery_monitoring/tasks.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from django.conf import settings
+
+from corehq.celery_monitoring.heartbeat import Heartbeat
+
+
+# Create one periodic_task named heartbeat__{queue} for each queue
+for queue, time_to_start_alert_threshold in settings.CELERY_HEARTBEAT_THRESHOLDS.items():
+    heartbeat = Heartbeat(queue)
+    locals()[heartbeat.periodic_task_name] = Heartbeat(queue).periodic_task
```

Comment:
nit, I think you could use `heartbeat.periodic_task` here

Reply:
You mean using `heartbeat.periodic_task.__name__` instead of `heartbeat.periodic_task_name`?

Reply:
no, I was just wondering if it was necessary to make another instance of the `Heartbeat` class here.

Reply:
Oh! Thanks, totally missed that. Will fix